app/javascript/elm/src/Main.elm

```diff
@@ -871,19 +871,11 @@ setScrollPosition value =
 viewDocument : Model -> Browser.Document Msg
 viewDocument model =
     { title = "AirCasting"
-    , body = [ view model ]
+    , body = [ snippetGoogleTagManager 
```

Comment:
imo it's ok to do it this way cos it's clearly visible that this element has to be the first thing rendered in the `body` so 👍